### ISSUE TITLE
WebM support in comments

### ISSUE THREAD

**Issue by aldipower** [isaacs/github#373]
Would be nice to be able to upload .webm's as easy as the other media, like pngs and gifs into a comment area.

**Comment by izidormatusov** [comment]
:+1:

**Comment by Apercu** [comment]
And support in READMEs too ;)

**Comment by kfrz** [comment]
Is there any progress on embedding webm into Github comments?

**Comment by Davilink** [comment]
Any news ? Did someone send a ticket to the GutHub support  ?

**Comment by bierik** [comment]
👍

**Comment by epatpol** [comment]
Almost 3 years later and still very needed I think! Webms are also much more lightweight and efficient than gifs.

**Comment by Davilink** [comment]
I send a ticket, wait and see for the answer.

**Comment by Davilink** [comment]
Here the response : 

> Hi
>
>Thanks for your feedback! We're always working to improve GitHub and we consider every suggestion we receive. I see that we have received past request to add WebM support so I have add you as a +1 to this request.
>
>We can't say if/when we may add a feature, however your feedback has definitely been recorded.
>
>Thanks,
>
>GitHub Support

**Comment by BoQsc** [comment]
Why WebM is still unsupported? it has been a long time.

**Comment by flightcrank** [comment]
its not rocket science, just add webM support already.

**Comment by balthazar** [comment]
@flightcrank That's not how you talk if you want something out of someone you don't know. Not commenting would have served the cause better.

To Github employees, I think this really should be prioritized considering not only the better quality, bandwidth reduction as suggested by @Fusl, but also for your storage size when gifs are uploaded directly.

**Comment by flightcrank** [comment]
@balthazar Simmer down now. Maybe you should't have commented either. At least i was short and to the point, unlike yourself.

**Comment by hediet** [comment]
Browser Extensions or even github itself could integrate https://recordscreen.io/ to record webm files directly from the browser.
This would make getting reproducible bug reports much easier!

**Comment by Davilink** [comment]
@hediet This is a different issue. The current one is to add support, in the readme preview, the use of webm file instead of simple image file.
Current Behavior:
![image](https://user-images.githubusercontent.com/1498932/95226986-8ec59100-07cb-11eb-8896-f9ff21deb6be.png)
What is expected:
![image](https://user-images.githubusercontent.com/1498932/95228438-51620300-07cd-11eb-8c38-a35e72154595.png)

**Comment by hediet** [comment]
@Davilink this issue is about supporting webm videos in comments, not markdown files. Support for webm files is necessary for such a screenrecorder (gif files would work too, but they become very large quickly). The screenrecorder itself could be implemented through a browser extension.

**Comment by Davilink** [comment]
@hediet yeah... my bad. But i still think this should be a separate issue. This issue is for supporting the WebM in Comment section (and someone after did propose to add it into the Readme https://github.com/isaacs/github/issues/373#issuecomment-152352854, but maybe this should be in separate issue as well).
The way i see it it like 3 story with one depending on one
- Add WebM support in comment section
- Add Screen Recording functionality (this was you suggesting) (depending on the "Add WebM support in comment section")
- Add WebM support in Readme

**Comment by y0nd0** [comment]
WebM videos are smaller with a better framerate. Especially with complex frames and duration. In my case, a few seconds, up to a minute: Smaller than 1 MB with a moderate quality. 

To embed the video, you have to mute the sound. Othweise, autoplay will not work. If you want to achieve a gif-like behavior. ...

**Comment by Fusl** [comment]
Video support is now partly implemented. I haven't tested with .webm's but .mp4 and .mov files are supposed to work:

![image](https://user-images.githubusercontent.com/2349496/102812943-dff32300-43bf-11eb-9fa3-bf0f4e2ccb50.png)

**Comment by balthazar** [comment]
@Fusl what's the relevance of your comment to this issue then?

**Comment by milesfrain** [comment]
Just tried uploading `.webm` in this comment. Still unsupported.
![image](https://user-images.githubusercontent.com/3578681/102813925-8f0f0a80-437e-11eb-985b-4e289b5d998b.png)

**Comment by BoQsc** [comment]
> Just tried uploading `.webm` in this comment. Still unsupported.
> ![image](https://user-images.githubusercontent.com/3578681/102813925-8f0f0a80-437e-11eb-985b-4e289b5d998b.png)

Same. 
WebM is still unsupported in GitHub.

**Comment by flightcrank** [comment]
who the hell uses MOV? WebM is the logical choice to support. But its the one thing that isnt supported. Typical microsoft.

**Comment by paulkolbovich** [comment]
renaming WebM to mp4 works for me as a temporary solution

```md
[!(renamed webm)](https://user-images.githubusercontent.com/294989/111512737-20650180-8761-11eb-80c5-fe717dc5014a.mp4)
```

[!(renamed webm)](https://user-images.githubusercontent.com/294989/111512737-20650180-8761-11eb-80c5-fe717dc5014a.mp4)
